modules/samples/artifacts/PublishJmsInKeyvalueFormat/PublishJmsInKeyvalueFormat.siddhi

```diff
@@ -0,0 +1,64 @@
+@App:name("PublishJmsInKeyvalueFormat")
```

Comment:
Convert the description as the following
https://raw.githubusercontent.com/wso2/product-sp/b62da269927f3288847b6564429bc378a28977c0/modules/samples/artifacts/PublishKafkaInJsonFormat/PublishKafkaInJsonFormat.siddhi